Dangerfile.js

```diff
@@ -0,0 +1,39 @@
+const { customRules } = require("@pagopa/danger-plugin");
+
+const recordScope = {
```

Comment:
Why are there so many scopes? How does handler-kit make use of them?

Reply:
if there is no JIRA project associated with this plugin we can also remove them!

Reply:
For now, we can use the "io-sign" JIRA board

Reply:
In my opinion JIRA should be mandatory, because tracking every activity on JIRA is our "engineering standard" that we need to strengthen